test/metabase/api/permissions_test.clj

```diff
@@ -116,14 +118,15 @@
 (deftest update-perms-graph-test
   (testing "PUT /api/permissions/graph"
     (testing "make sure we can update the perms graph from the API"
-      (mt/with-temp PermissionsGroup [group]
-        (mt/user-http-request
-         :crowberto :put 200 "permissions/graph"
-         (assoc-in (perms/data-perms-graph)
-                   [:groups (u/the-id group) (mt/id) :data :schemas]
-                   {"PUBLIC" {(mt/id :venues) :all}}))
-        (is (= {(mt/id :venues) :all}
-               (get-in (perms/data-perms-graph) [:groups (u/the-id group) (mt/id) :data :schemas "PUBLIC"]))))
+      (let [db-id (mt/id :venues)]
```

Comment:
This is ugly. If I inline `db-id`, the tests fails because the starting permission graph is generated without this database, but the endpoint already sees it. As a result, the missing admin group permissions for it are seen as a diff causing an error.